autoapi/mappers/dotnet.py

```diff
@@ -49,23 +49,35 @@ class DotNetSphinxMapper(SphinxMapperBase):
     '''Auto API domain handler for .NET
 
     Searches for YAML files, and soon to be JSON files as well, for auto API
-    sources
+    sources. If not pattern configuration was explicitly specified, then default
+    to looking up a ``docfx.json`` file.
 
     :param app: Sphinx application passed in as part of the extension
     '''
 
     top_namespaces = {}
 
+    DOCFX_OUTPUT_PATH = '_api'
+
     def load(self, patterns, dirs, ignore=None, **kwargs):
-        '''
-        Load objects from the filesystem into the ``paths`` dictionary.
+        '''Load objects from the filesystem into the ``paths`` dictionary.
 
+        If a patterns setting was not specified, look for a ``docfx.json`` file
+        by default.  A ``docfx.json`` should be treated as the canonical source
+        before the default patterns.  Fallback to default pattern matches if no
+        ``docfx.json`` files are found.
```

Comment:
Used first, 'if a patterns setting was not specified'. This could be clearer though.